src/godot-utils.ts

```diff
@@ -13,3 +15,27 @@ export function fixPathForGodot(path: string): string {
 
     return path;
 }
+
+export async function findGodotExecutablePath(): Promise<string | undefined>
+{
+    let path: string | undefined;
+
+    // If the user has set the path in the settings, use that value
+    path = Configuration.Value.godotExecutablePath;
+    if (path)
+    {
+        return path;
+    }
+
+    // Check if the godot command is in the path
+    path = await lookpath('godot');
+    if (path) {
+        return path;
+    }
+
+    // TODO: How to find the Godot executable?
+    // We could check common paths...
```

Comment:
If the extension is connected to a running Godot editor instance, then it should be possible to get the Godot editor path from the metadata:

https://github.com/godotengine/godot-csharp-vscode/blob/3de6294befff33c5c5e3bf302a126b27c0927b1b/src/godot-tools-messaging/client.ts#L216
https://github.com/godotengine/godot-csharp-vscode/blob/3de6294befff33c5c5e3bf302a126b27c0927b1b/src/godot-tools-messaging/client.ts#L388

In fact, this function should always return that before anything else if `Configuration.Value.godotExecutablePath` is not set.

Reply:
To be clear, I'm not requesting to implement this change in this PR. Just giving one of the answers to the question in the comment. It's probably a good idea to include this info in the comment, though.

Reply:
Done